editor/editor_node.cpp

```diff
@@ -7305,7 +7305,18 @@ EditorNode::EditorNode() {
 	}
 
 	remove_android_build_template = memnew(ConfirmationDialog);
-	remove_android_build_template->set_ok_button_text(TTR("Show in File Manager"));
+
+#if defined(WINDOWS_ENABLED)
+	const String ok_button_text = TTR("Reveal in File Explorer");
+#elif defined(MACOS_ENABLED)
+	const String ok_button_text = TTR("Reveal in Finder");
+#elif defined(IOS_ENABLED)
+	const String ok_button_text = TTR("Reveal in Files");
```

Comment:
Why this? This won't be used without iOS editor support and AFAIK that's blocked by limitations with the platform itself, with the needed capabilities and ability to distribute the editor

Reply:
You can technically build the iOS editor if you want. So, it should be supported, especially if there's no drawbacks.

Reply:
Sure but I don't see the value in adding it to something that doesn't have official support

Reply:
Yes, you can build editor for iOS (with some code changes and a lot of limitations), but `shell_show_in_file_manager` is not implemented for iOS, so this editor feature will not work.

Reply:
I'll revert then the latest commit.